core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponent.scala

```diff
@@ -481,11 +482,11 @@ trait EntityComponent {
       val entitiesGrouped = entities.grouped(batchSize).toSeq
 
       DBIO.sequence(entitiesGrouped map { batch =>
-        EntityRecordRawSqlQuery.action(workspaceContext.workspaceId, batch.map(r => AttributeEntityReference(r.entityType, r.name)))
+        EntityRecordRawSqlQuery.action(workspaceContext.workspaceId, batch.map(_.toReference))
       }).map(_.flatten)
     }
 
-    def cloneEntities(destWorkspaceContext: SlickWorkspaceContext, entities: TraversableOnce[Entity]): ReadWriteAction[Unit] = {
+    def cloneEntities(destWorkspaceContext: SlickWorkspaceContext, entities: TraversableOnce[Entity]): ReadWriteAction[Iterator[Option[Int]]] = {
```

Comment:
not that I really care so much, but why is this better? IMO the return value has no meaning and exposes irrelevant details about the inner workings of this function. This goes for the change on line 17 too.